### PR TITLE
Creation date for encounters and obs are now set on the back-end on the forms framework

### PR DESCRIPTION
**Pull request by edrisse**

### CODE REVIEW

**Comment by ynurmahomed** [comment]
[Let back-end determine creation date](https://trello.com/c/ZCKAhOtb/66-let-back-end-determine-creation-date)

File: app/poc-common/clinical-services/mappers/createEncounterMapper.js [hunk 1]
```diff
@@ -64,7 +63,6 @@
         if (field.field.concept.set) {
           var obs = {
             concept: field.field.concept.uuid,
-            obsDatetime: Bahmni.Common.Util.DateUtil.now(),
```
Comment (valvijo) [inline app/poc-common/clinical-services/mappers/createEncounterMapper.js]: What if the date is passed by the user in retrospective mode, test request and drug prescription, lab and pharmacy modules respectively?

Comment (edrisse) [reply app/poc-common/clinical-services/mappers/createEncounterMapper.js]: Those parts of the application do not use the createEncounterMapper right? We are planning to address that on a future PR, when the proper back-end support is added.

Comment (Guimino) [reply app/poc-common/clinical-services/mappers/createEncounterMapper.js]: @valvijo  and @edrisse , I have done these changes for pharmacy module , I'm just creating some unit tests, by the end of the day will commit these changes

Comment (valvijo) [reply app/poc-common/clinical-services/mappers/createEncounterMapper.js]: Got it

**Pull request merged**